src/mlpack/methods/ne/neat.hpp

```diff
@@ -713,32 +717,19 @@ class NEAT {
     }
 
     // Keep the best in each species.
-
-    //DEBUGGING!!!!!!!!!
-    printf("before, species sizes are: ");
-    for (ssize_t s=0; s<aPopulation.aSpecies.size(); ++s) {
-      std::cout<< aPopulation.aSpecies[s].aGenomes.size() << "  ";
-    }
-    printf("\n");
-    //DEBUGGING!!!!!!!!!
-
     CullSpeciesToOne(aPopulation);  // NOTICE: I found that this makes XOR test converges fast.
 
-    //DEBUGGING!!!!!!!!!
-    printf("after, species sizes are: ");
-    for (ssize_t s=0; s<aPopulation.aSpecies.size(); ++s) {
-      std::cout<< aPopulation.aSpecies[s].aGenomes.size() << "  ";
-    }
-    printf("\n");
-    //DEBUGGING!!!!!!!!!
-
     // Random choose species and breed child until reach population size.
     childGenomes.push_back(lastBestGenome);
```

Comment:
ah, right I thought the lastBestGenome is a reference but it's a copy, so it does exist.